Jenkinsfile

```diff
@@ -135,6 +140,12 @@ def buildStep(target, compilerLabel, toolchain, radioShield, meshInterface) {
             execute("sed -i 's/\"mbed-mesh-api.6lowpan-nd-panid-filter\": \"0xffff\"/\"mbed-mesh-api.6lowpan-nd-panid-filter\": \"0xABBA\"/' ${config_file}")
           }
 
+          if ("${meshInterface}" == "ws") {
+            config_file = "./configs/mesh_wisun${config_suffix}.json"
+            // Possibly in future use systest Wi-SUN Border Router for testing (Network name = "ARM-WS-TESTING")
+            execute("sed -i 's/\"mbed-mesh-api.wisun-network-name\": \"\\\"WS-TEST\\\"\"/\"mbed-mesh-api.wisun-network-name\": \"\\\"ARM-WS-TESTING\\\"\"/' ${config_file}")
```

Comment:
Fix Jenkinsfile to replace "Wi-SUN Network" with "ARM-WS-LAB-NWK"

Reply:
Done